lib/adapters/session_adapter.rb

```diff
@@ -22,9 +33,30 @@ def folder_selected?
 
   private
 
-  def session_file
+  def extracted_token(token)
+    return {} unless token
+
+    decoded_token = Base64.decode64(token)
+    attrs = decoded_token.split(';')
+    {
+      username: attrs.first,
+      token: attrs[1]
```

Comment:
```suggestion
      token: attrs.second
```
So wege consistency. Odr haut beidi [0] und [1]